src/rpc/mining.cpp

```diff
@@ -248,9 +248,11 @@ UniValue getmininginfo(const UniValue& params, bool fHelp)
             "  \"errors\": \"...\"          (string) Current errors\n"
             "  \"generate\": true|false     (boolean) If the generation is on or off (see getgenerate or setgenerate calls)\n"
             "  \"genproclimit\": n          (numeric) The processor limit for generation. -1 if no generation. (see getgenerate or setgenerate calls)\n"
+            "  \"networkhashps\": n         (numeric) An estimate of the number of hashes per second the network is generating to maintain the current difficulty\n"
             "  \"pooledtx\": n              (numeric) The size of the mem pool\n"
             "  \"testnet\": true|false      (boolean) If using testnet or not\n"
             "  \"chain\": \"xxxx\",         (string) current network name as defined in BIP70 (main, test, regtest)\n"
+            "  \"generate\": \"xxxx\",      (boolean) True if generation currently enabled\n"
```

Comment:
? see above, line 249

Reply:
Updated. Think I did a copy/paste instead of a cut/paste.

Reply:
Hmmm... the text of the new description is quite different actually and doesn't follow the rules for `boolean` - `\"xxxx\"` instead of `true|false`. Why is that?

Reply:
Yes, you're right. I am not sure how I did that. I will update it to match what was there originally (but put it in the right order).